src/main/java/com/softserve/edu/sporthubujp/entity/Article.java

```diff
@@ -10,11 +10,7 @@
 
 
 @Entity
-@Getter
-@Setter
-@EqualsAndHashCode
-@ToString
-@AllArgsConstructor
+@Data
```

Comment:
Exclude all @OneToMany from equals and hashCode, because later you will get LazyInitializationException